src/test/scala/org/apache/spark/shuffle/RssShuffleManagerWithServerSequenceRegistryTest.scala

```diff
@@ -0,0 +1,133 @@
+/*
+ * This file is copied from Uber Remote Shuffle Service
+(https://github.com/uber/RemoteShuffleService) and modified.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.shuffle
+
+import org.apache.spark._
+import org.apache.spark.executor.{ShuffleWriteMetrics, TempShuffleReadMetrics}
+import com.uber.rss.testutil.TestStreamServer
+import org.scalatest.Assertions._
+import org.testng.annotations._
+
+import java.util.UUID
+import scala.collection.mutable.ArrayBuffer
+
+
+class RssShuffleManagerWithServerSequenceRegistryTest {
+
+  var appId: String = null
+
+  var sc: SparkContext = null
+
+  var testStreamServer: TestStreamServer = null
+  private var shuffleManagers = ArrayBuffer[RssShuffleManager]()
+
+  @BeforeMethod
+  def beforeTestMethod(): Unit = {
+    appId = UUID.randomUUID().toString()
+    shuffleManagers.clear()
+
+    testStreamServer = TestStreamServer.createRunningServerWithLocalStandaloneRegistryServer(appId)
+  }
+
+  @AfterMethod
+  def afterTestMethod(): Unit = {
+    sc.stop()
+    shuffleManagers.foreach(m => m.stop())
+    testStreamServer.shutdown()
+  }
+
+  @Test
+  def runSparkApplication(): Unit = {
+    val conf = TestUtil.newSparkConfWithStandAloneRegistryServer(appId, "")
+    conf.set("spark.shuffle.rss.serviceRegistry.type", "serverSequence")
+    conf.set("spark.shuffle.rss.serverSequence.connectionString",
+      s"localhost:${testStreamServer.getShufflePort}")
+    conf.set("spark.shuffle.rss.serverSequence.serverId", testStreamServer.getServerId)
```

Comment:
maybe we could land PR #45 first and use `newSparkConfWithSequenceServerRegistryServer` method to initialize the test server.

Reply:
> maybe we could land PR #45 first and use `newSparkConfWithSequenceServerRegistryServer` method to initialize the test server.

This has to happen this way as we need to create a dummy registry and then let client know the service registry to use the s tatic list for the servers in K8